config/routes.d/staging.horse.json

```diff
@@ -5,7 +5,8 @@
       "^/docs/private-cloud/": "developer.rackspace.com/user-guide.html",
       "^/docs/api-doc-template/": "developer.rackspace.com/user-guide.html",
       "^/docs/billing-api/": "developer.rackspace.com/user-guide.html",
-      "^/docs/sdp/": "developer.rackspace.com/user-guide.html"
+      "^/docs/sdp/": "developer.rackspace.com/user-guide.html",
+      "^/docs/rpc-vmware/server-virt-customer-handbook/":"developer.rackspace.com/server-virt-customer-handbook.html"
```

Comment:
@asettle, so the value of the key:value pair here should be the template that deconst will use to render the pages. I think you want `"developer.rackspace.com/user-guide.html"` unless you intend to create a template, too.